### PR TITLE
Adding OCI8 Support

### PR DESCRIPTION
**Pull request by mixaster**
Only PHP 8.0 is supported by the extension currently

`pecl/oci8 requires PHP (version >= 8.0.0)`

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -38,37 +38,38 @@ functions:
...
-| Redis              | `${bref-extra:redis-php-74}`          |
-| Scrypt             | `${bref-extra:scrypt-php-74}`         |
-| SPX                | `${bref-extra:spx-php-74}`            |
-| Symfony Runtime    | `${bref-extra:symfony-runtime-php-74}`|
-| Microsoft SQLSRV   | `${bref-extra:sqlsrv-php-74}`         |
-| UUID               | `${bref-extra:uuid-php-74}`           |
-| Xdebug             | `${bref-extra:xdebug-php-74}`         |
-| Yaml               | `${bref-extra:yaml-php-74}`           |
+| Name             | Serverless config (php 7.4)            |
+| :--------------- | :------------------------------------- |
```
Comment (Nyholm) [inline Readme.md:42]: Please only insert the `Oracle` row. Leave the other CS fixes unchanged.

**Review by Nyholm — COMMENTED** [review]
Thank you, this looks good. Just revert the CS fixes in the table and I'll be happy to merge.

**Comment by Nyholm** [comment]
Hey. Just a small update and I'll be happy to merge this PR.

**Review by KaivanOmmeren — APPROVED** [review]

**Comment by KaivanOmmeren** [comment]
This PR is exactly what I was looking for! Hopefully i'll be able to use this soon :)

**Comment by Nyholm** [comment]
Yes. As soon as @mixaster reverts the CS changes I'll be happy to merge.

**Comment by mixaster** [comment]
Sorry about that! I didn't see the notification. I see the other PR pulled in the changes, so it's fine to close this one.

**Comment by Nyholm** [comment]
No worries. I've merged the other PR now. 
I'm just preparing a release.

**Pull request closed without merging**